tests/integration/targets/azure_rm_image/tasks/setup_des.yml

```
@@ -0,0 +1,96 @@
+- name: Set DES variables
+  ansible.builtin.set_fact:
+    des_name: "{{ 'des' ~ uid_short }}"
+    uid_short: "{{ (resource_group ~ inventory_hostname) | hash('md5') | truncate(10, True, '') }}"
+    vault_name: "kv{{ uid_short }}{{ '%m%d%H%M%S' | strftime }}"
```

This cannot be used directly when setting parameters
```suggestion
    vault_name: "kv{{ resource_group | hash('md5') | truncate(8, True, '') }}{{ '%m%d%H%M%S' | strftime }}"
```

---

Fixed.